awslimits/manage.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sendgrid
+from flask_script import Manager
+
+from server import app
+from support import get_limits_for_alert, alert_email_body
+
+manager = Manager(app)
+
+
+@manager.command
+def send_alerts():
+	limits_for_alert = get_limits_for_alert()
+	if limits_for_alert:
+		content = alert_email_body(limits_for_alert)
+		email_recipients = [{'email': email} for email in os.environ.get('EMAIL_RECIPIENTS').split(',')]
+		sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
```

Comment:
Then we probably should ensure that exists before we start the application, like we do for yaws_web (`assert os.environ.get('SENDGRID_API_KEY')` or a variant). A `settings` file or object wouldn't be a bad idea also.